lms/djangoapps/discussion_api/api.py

```diff
@@ -415,7 +415,7 @@ def get_comment_list(request, thread_id, endorsed, page, page_size, mark_as_read
         raise Http404
     num_pages = (resp_total + page_size - 1) / page_size if resp_total else 1
 
-    results = [CommentSerializer(response, context=context).data for response in responses]
+    results = [CommentSerializer(response, remove_fields=["children"], context=context).data for response in responses]
```

Comment:
@nasthagiri @jcdyer 
interestingly `recursive` has no impact on CS response; I am new at Ruby but as far as I could understand the code, `recursive` is not even being used by this endpoint. The CS uses `merge_response_content` to make response list and it always includes child comments; no argumented option available to exclude children so far.
@BenjiLee Please correct me if i misunderstood anything

Do we update comment service to be able to optionally include children comment?